lib/src/components/NxThreatCounter/NxThreatCounter.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-threat-counter {
+  align-items: center;
+  box-sizing: content-box;
+  border: 1px solid;
+  border-left-width: 8px;
+  border-radius: $nx-border-radius;
+  display: inline-flex;
+  font-size: $nx-font-size-s;
+  height: 30px;
+  margin: 0;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+}
+
+.nx-threat-counter__count {
+  @include bold();
+}
+
+.nx-threat-counter__text {
+  @include semi-bold();
+  margin-left: $nx-spacing-xs;
+  text-transform: uppercase;
+}
+
+.nx-threat-counter--critical {
+  border-color: $nx-threat-color-critical;
+}
+
+.nx-threat-counter--severe {
+  border-color: $nx-threat-color-severe;
+}
+
+.nx-threat-counter--moderate {
+  border-color: $nx-threat-color-moderate;
+}
+
+.nx-threat-counter--low {
+  border-color: $nx-threat-color-low;
+}
+
+.nx-threat-counter-container {
+  display: inline-grid;
+  row-gap: $nx-spacing-md;
+  column-gap: $nx-spacing-md;
+  grid-template-columns: auto auto auto auto;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-threat-counter-container--column {
+  grid-template-rows: auto auto;
+  grid-template-columns: auto;
+}
+
+.nx-threat-counter-container--grid {
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
```

Comment:
Given that the number of items is constant and `grid-template-columns` is set, `grid-template-rows` is redundant.  Also the value for it here in `.nx-thread-counter-container--column` doesn't make sense